### PR TITLE
EASYOPAC-1284 - Narrow module to only work for module.

### PR DESCRIPTION
**Pull request by guddo**
https://inlead.atlassian.net/browse/EASYOPAC-1284

This PR makes sure that this module only "hits" the actual needed CSS, and not any given CSS with the same selector name.

### CODE REVIEW

File: ding_search_collection.module [hunk 1]
```diff
@@ -8,10 +8,15 @@
  * Implements hook_entity_view().
  */
 function ding_search_collection_entity_view($entity, $entity_type, $view_mode, $langcode) {
+  if ($entity_type == 'ting_collection' && $view_mode == 'minimal') {
+    return;
+  }
+
   // Apply only to ting_collection teaser.
```
Comment (guddo) [inline ding_search_collection.module:14]: Why is this needed, when the above `if` is already in place?

Comment (OIOIOOOI) [reply ding_search_collection.module:14]: I would expand this with the idea that all three below conditional can be minimized:
```
if ($entity_type == 'ting_collection' && $view_mode == 'minimal') {
    return;
  }

  // Apply only to ting_collection teaser.
  if ($entity_type != 'ting_collection' && $view_mode != 'teaser') {
    return;
  }

  // Do not show on collection page.
  if (arg(0) == 'ting' && arg(1) == 'collection') {
    return;
  }```

**Review by OIOIOOOI — APPROVED** [review]
